detectable/src/main/java/com/synopsys/integration/detectable/detectables/go/gomod/GoModGraphTransformer.java

```diff
@@ -0,0 +1,55 @@
+package com.synopsys.integration.detectable.detectables.go.gomod;
+
+import java.io.File;
+import java.util.List;
+import java.util.Map;
+
+import com.google.gson.Gson;
+import com.synopsys.integration.blackduck.service.BlackDuckServicesFactory;
+import com.synopsys.integration.detectable.detectable.exception.DetectableException;
+import com.synopsys.integration.detectable.detectable.executable.ExecutableRunnerException;
+
+public class GoModGraphTransformer {
```

Comment:
Perhaps change this to GoModGraphGenerator to be more inline with GoListUJsonGenerator? Or vice versa. They both execute go mod commands and put that data in a useable format.

Reply:
Hmm I see your point.  I thought there might be enough of a distinction in that GoListUJsonGenerator doesn't change the output of the go mod command, it just does work to decide which command to execute, whereas GoModGraphTransformer gets the command output and then makes changes.  I'm okay calling them both generators if you disagree.

Reply:
Okay so what if you had one class called GoModCommandExecutor that did all the executing of commands, and this class would just do the transformation.